doc/users/next_whats_new/label_outer.rst

```diff
@@ -0,0 +1,26 @@
+Remove inner ticks in ``label_outer()``
+---------------------------------------
+Up to now, ``label_outer()`` has only removed the ticklabels. The ticks lines
+were left visible. This is now configurable through a new parameter
+``label_outer(remove_inner_ticks=True)``.
+
+
+.. plot::
```

Comment:
This probably needs an example in the docs somewhere as well?    Perhaps also mention at https://matplotlib.org/devdocs/users/explain/axes/axes_ticks.html#styling-ticks-tick-parameters?

Reply:
I plan to use it in #24379